students/sbolsen/lesson03/slice.py

```diff
@@ -0,0 +1,33 @@
+def first_last_exchanged(sequence):
+    sequence[0], sequence[-1] = sequence[-1], sequence[0]
+    return sequence
```

Comment:
this is technically not slicing 🙂 

slicing is `[<start>:<stop>:<step>]`

so how could you do this with slicing only? (hint: it's a one liner)